src/SFA.DAS.Roatp.CourseManagement.Web.UnitTests/Controllers/ProviderLocationsControllerTests.cs

```diff
@@ -0,0 +1,113 @@
+﻿using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.Roatp.CourseManagement.Application.ProviderLocations.Queries;
+using SFA.DAS.Roatp.CourseManagement.Domain.ApiModels;
+using SFA.DAS.Roatp.CourseManagement.Web.Controllers;
+using SFA.DAS.Roatp.CourseManagement.Web.Infrastructure.Authorization;
+using SFA.DAS.Roatp.CourseManagement.Web.Models.ProviderLocations;
+using System;
+using System.Security.Claims;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.Roatp.CourseManagement.Web.UnitTests.Controllers
+{
+    [TestFixture]
+    public class ProviderLocationsControllerTests
```

Comment:
Create a folder for PrvoviderLocationsControllerTests and then add a class for method GetProvidersTrainingLocations using name GetProvidersTrainingLocationsTest